### ISSUE TITLE
[BUG] Sound Blaster Pro v1 installation bug and FM music not playing

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#497]
The installer software disk of SB pro v1 does not find the card at the port 220 (which is the default).
If I disable the OPL emulation option the installer works fine, The installed Windows 3.1 does not detect the card correctly if the  OPL chip is not enabled.
When I enabled again the OPL option but the DOS diagnostic utility does not play FM music.  I however it plays the 8-bit voice testing correctly.
All works flawlessly if I configure the card as Sound blaster Pro v2.

I'm using the last version of 86box on Windows 10.
Here is the image file of the hard disk.

[https://drive.google.com/open?id=1XNPCMR-qE1CTDDLjlCuuxNFtCE9HwRux](url)

**Comment by OBattler** [comment]
Can you please upload 86box.cfg and the nvr folder?

**Comment by Fenix770** [comment]
I found the problem. The drivers are for the SB Pro v2.0.
After installing the drivers, The installation disks are named "SB pro" but I later discovered that they were for the SB pro v2. The installed midi mapper in Windows 3.1 uses OPL 3 chip instead Dual OPL 2. And the in the SET BLASTER the environment is specified by the setup as Type 4 (T4), in autoexec.bat , instead of T2 (for the SB Pro V1). 
I managed to find the Windows drivers for the SB Pro v1 and works correctly. The DOS, utilities are nearly the same except for one of two TSR drivers (like the SBFMDRV).
